diff --git a/services.py b/services.py
--- a/services.py
+++ b/services.py
@@ -1,25 +1,25 @@
-def method_one():
-    return "one_a"
-    return "one_b"
-    return "one_c"
-
-def method_two():
-    return "two_a"
-    return "two_b"
-    return "two_c"
-
-def method_three():
-    return "three_a"
-    return "three_b"
-    return "three_c"
-
-def method_four():
-    return "four_a"
-    return "four_b"
-    return "four_c"
-
-def method_five():
+def method_five():
     return "five_a"
-    return "five_b"
+    return "five_b_changed"
     return "five_c"
+
+def method_four():
+    return "four_a"
+    return "four_b"
+    return "four_c"
+
+def method_three():
+    return "three_a"
+    return "three_b"
+    return "three_c"
+
+def method_two():
+    return "two_a"
+    return "two_b"
+    return "two_c"
+
+def method_one():
+    return "one_a"
+    return "one_b"
+    return "one_c"
